tests/normalizer/mswordnormalizer.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+import MSWordNormalizer from '../../src/normalizer/mswordnormalizer';
+
+// `exec()` of the msword normalizer is tested with autogenerated normalization tests.
+describe( 'PasteFromOffice/normalizer/mswornormalizer', () => {
+	const normalizer = new MSWordNormalizer();
+
+	describe( 'isActive()', () => {
+		describe( 'correct data set', () => {
+			[
+				'<meta name=Generator content="Microsoft Word 15"><p>Foo bar</p>',
```

Comment:
It does not detect the other option - only one form of compatible content.

Reply:
Compare readability of the tests:

```js
describe( 'isActive()', () => {
	it( 'should return true for MS Word content', () => {
		expect( normalizer.isActive( '<meta name=Generator content="Microsoft Word 15"><p>Foo bar</p>' ) ).to.be.true;
	} );

	it( 'should return true for MS Word content - in Safari', () => {
		expect( normalizer.isActive( '<meta name=Generator content="Microsoft Word 15"><p>Foo bar</p>' ) ).to.be.true;
	} );

	it( 'should return false for non-compatible content', () => {
		expect( normalizer.isActive( '<p>Foo bar</p>' ) ).to.be.false;
	} );

	it( 'should return false for content from other source', () => {
		expect( normalizer.isActive( '<p id="docs-internal-guid-12345678-1234-1234-1234-1234567890ab"></p>' ) ).to.be.false;
	} );
} );
```

marking every test with numbers provides little value for others - I have to check what the `#` was and why it might fail.

Those tests are simple enough and doesn't have to be run in a loop.

Reply:
> It does not detect the other option - only one form of compatible content.

It wasn't present such check in original data. Now it's added :)